tests/tests/src/main/scala/tests/markdown/Compat.scala

```diff
@@ -3,46 +3,81 @@ package tests.markdown
 import tests.BuildInfo
 
 object Compat {
+  sealed trait ScalaVersion
+  case object Scala2 extends ScalaVersion
+  case object Scala211 extends ScalaVersion
+  case object Scala212 extends ScalaVersion
+  case object Scala213 extends ScalaVersion
+  case object Scala3 extends ScalaVersion
+  case class Full(ver: String) extends ScalaVersion
+  case object All extends ScalaVersion
+
+  private def isCurrent(ver: ScalaVersion): Boolean = {
+    val full = BuildInfo.scalaVersion
+    val binary = BuildInfo.scalaBinaryVersion
+
+    ver match {
+      case Scala2 => binary.startsWith("2.")
+      case Scala211 => full.startsWith("2.11")
+      case Scala212 => full.startsWith("2.12")
+      case Scala213 => full.startsWith("2.13")
+      case Scala3 => full.startsWith("3.0") || binary == "3"
+      case Full(v) => v == full
+      case All => true
+    }
+  }
+
+  private def binary: ScalaVersion = {
```

Comment:
This doesn't seem to be used.

Reply:
Ugh, github lost a comment - it's used further down: https://github.com/scalameta/mdoc/pull/504/files#diff-27c1ab918805d95f9c6d6f3687da7a835633ec32bb87b4f41923411f7d9b3791R47